spec/requirefire.spec.ts

```diff
@@ -16,9 +17,11 @@ describe("requirefire", () => {
   });
 
   it("should not use node's built in require to require modules", () => {
-    const fired = _require("./fixtures/mod_a");
-    const normal = require("./fixtures/mod_a");
-    expect(fired).not.toBe(normal);
+    jest.isolateModules(() => {
```

Comment:
Sprinkled a few of these around so that the require cache that jest maintains doesn't get "tainted"